snippets/capabilities-table.adoc

```diff
@@ -8,16 +8,16 @@ The following table describes the `baselineCapabilitySet` values.
 |Value|Description
 
 |`vCurrent`
-|Specify when you want to automatically add new capabilities as they become recommended.
+|Specify when you want to automatically add new capabilities as they become available.
```

Comment:
"available" isn't quite right here.  Maybe "as the default capabilities evolve with new releases" or something?